[2023-10-17T11:19:32.835+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-05-02T00:00:00+00:00 [queued]>
[2023-10-17T11:19:32.846+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-05-02T00:00:00+00:00 [queued]>
[2023-10-17T11:19:32.846+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2023-10-17T11:19:32.858+0000] {taskinstance.py:1382} INFO - Executing <Task(FileSensor): waiting_for_data> on 2021-05-02 00:00:00+00:00
[2023-10-17T11:19:32.867+0000] {standard_task_runner.py:57} INFO - Started process 678 to run task
[2023-10-17T11:19:32.870+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'copy_tranform_to_mongo', 'waiting_for_data', 'scheduled__2021-05-02T00:00:00+00:00', '--job-id', '126', '--raw', '--subdir', 'DAGS_FOLDER/file_***-mongo.py', '--cfg-path', '/tmp/tmpl1m8ue4j']
[2023-10-17T11:19:32.871+0000] {standard_task_runner.py:85} INFO - Job 126: Subtask waiting_for_data
[2023-10-17T11:19:32.914+0000] {task_command.py:416} INFO - Running <TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-05-02T00:00:00+00:00 [running]> on host 238c1d4ff767
[2023-10-17T11:19:33.005+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='copy_tranform_to_mongo' AIRFLOW_CTX_TASK_ID='waiting_for_data' AIRFLOW_CTX_EXECUTION_DATE='2021-05-02T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2021-05-02T00:00:00+00:00'
[2023-10-17T11:19:33.016+0000] {base.py:73} INFO - Using connection ID 'fs_default' for task execution.
[2023-10-17T11:19:33.017+0000] {filesystem.py:64} INFO - Poking for file /opt/in/tiktok_google_play_reviews.csv
[2023-10-17T11:19:33.017+0000] {filesystem.py:69} INFO - Found File /opt/in/tiktok_google_play_reviews.csv last modified: 20220405221612
[2023-10-17T11:19:33.017+0000] {base.py:287} INFO - Success criteria met. Exiting.
[2023-10-17T11:19:33.024+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=copy_tranform_to_mongo, task_id=waiting_for_data, execution_date=20210502T000000, start_date=20231017T111932, end_date=20231017T111933
[2023-10-17T11:19:33.082+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-10-17T11:19:33.101+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-10-18T10:43:06.645+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-05-02T00:00:00+00:00 [queued]>
[2023-10-18T10:43:06.653+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-05-02T00:00:00+00:00 [queued]>
[2023-10-18T10:43:06.653+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2023-10-18T10:43:06.665+0000] {taskinstance.py:1382} INFO - Executing <Task(FileSensor): waiting_for_data> on 2021-05-02 00:00:00+00:00
[2023-10-18T10:43:06.670+0000] {standard_task_runner.py:57} INFO - Started process 1640 to run task
[2023-10-18T10:43:06.673+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'copy_tranform_to_mongo', 'waiting_for_data', 'scheduled__2021-05-02T00:00:00+00:00', '--job-id', '125', '--raw', '--subdir', 'DAGS_FOLDER/file_***-mongo.py', '--cfg-path', '/tmp/tmpe50w77di']
[2023-10-18T10:43:06.674+0000] {standard_task_runner.py:85} INFO - Job 125: Subtask waiting_for_data
[2023-10-18T10:43:06.732+0000] {task_command.py:416} INFO - Running <TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-05-02T00:00:00+00:00 [running]> on host 2473b23ba069
[2023-10-18T10:43:06.815+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='copy_tranform_to_mongo' AIRFLOW_CTX_TASK_ID='waiting_for_data' AIRFLOW_CTX_EXECUTION_DATE='2021-05-02T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2021-05-02T00:00:00+00:00'
[2023-10-18T10:43:06.834+0000] {taskinstance.py:1937} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/sensors/base.py", line 257, in execute
    raise e
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/sensors/base.py", line 239, in execute
    poke_return = self.poke(context)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/sensors/filesystem.py", line 61, in poke
    hook = FSHook(self.fs_conn_id)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/hooks/filesystem.py", line 38, in __init__
    conn = self.get_connection(conn_id)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/hooks/base.py", line 72, in get_connection
    conn = Connection.get_connection_from_secrets(conn_id)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/connection.py", line 477, in get_connection_from_secrets
    raise AirflowNotFoundException(f"The conn_id `{conn_id}` isn't defined")
airflow.exceptions.AirflowNotFoundException: The conn_id `fs_default` isn't defined
[2023-10-18T10:43:06.838+0000] {taskinstance.py:1400} INFO - Marking task as FAILED. dag_id=copy_tranform_to_mongo, task_id=waiting_for_data, execution_date=20210502T000000, start_date=20231018T104306, end_date=20231018T104306
[2023-10-18T10:43:06.857+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 125 for task waiting_for_data (The conn_id `fs_default` isn't defined; 1640)
[2023-10-18T10:43:06.886+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 1
[2023-10-18T10:43:06.915+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-10-18T11:17:42.128+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-05-02T00:00:00+00:00 [queued]>
[2023-10-18T11:17:42.140+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-05-02T00:00:00+00:00 [queued]>
[2023-10-18T11:17:42.141+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2023-10-18T11:17:42.157+0000] {taskinstance.py:1382} INFO - Executing <Task(FileSensor): waiting_for_data> on 2021-05-02 00:00:00+00:00
[2023-10-18T11:17:42.163+0000] {standard_task_runner.py:57} INFO - Started process 551 to run task
[2023-10-18T11:17:42.165+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'copy_tranform_to_mongo', 'waiting_for_data', 'scheduled__2021-05-02T00:00:00+00:00', '--job-id', '206', '--raw', '--subdir', 'DAGS_FOLDER/file_***-mongo.py', '--cfg-path', '/tmp/tmp25ppfisj']
[2023-10-18T11:17:42.166+0000] {standard_task_runner.py:85} INFO - Job 206: Subtask waiting_for_data
[2023-10-18T11:17:42.209+0000] {task_command.py:416} INFO - Running <TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-05-02T00:00:00+00:00 [running]> on host 0e68aa52dfc7
[2023-10-18T11:17:42.288+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='copy_tranform_to_mongo' AIRFLOW_CTX_TASK_ID='waiting_for_data' AIRFLOW_CTX_EXECUTION_DATE='2021-05-02T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2021-05-02T00:00:00+00:00'
[2023-10-18T11:17:42.299+0000] {base.py:73} INFO - Using connection ID 'fs_default' for task execution.
[2023-10-18T11:17:42.301+0000] {filesystem.py:64} INFO - Poking for file /opt/in/tiktok_google_play_reviews.csv
[2023-10-18T11:17:42.301+0000] {filesystem.py:69} INFO - Found File /opt/in/tiktok_google_play_reviews.csv last modified: 20220405221612
[2023-10-18T11:17:42.302+0000] {base.py:287} INFO - Success criteria met. Exiting.
[2023-10-18T11:17:42.313+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=copy_tranform_to_mongo, task_id=waiting_for_data, execution_date=20210502T000000, start_date=20231018T111742, end_date=20231018T111742
[2023-10-18T11:17:42.378+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-10-18T11:17:42.408+0000] {taskinstance.py:2778} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2023-10-19T12:42:58.231+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-05-02T00:00:00+00:00 [queued]>
[2023-10-19T12:42:58.328+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-05-02T00:00:00+00:00 [queued]>
[2023-10-19T12:42:58.330+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2023-10-19T12:42:58.442+0000] {taskinstance.py:1382} INFO - Executing <Task(FileSensor): waiting_for_data> on 2021-05-02 00:00:00+00:00
[2023-10-19T12:42:58.479+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'copy_tranform_to_mongo', 'waiting_for_data', 'scheduled__2021-05-02T00:00:00+00:00', '--job-id', '221', '--raw', '--subdir', 'DAGS_FOLDER/file_***-mongo.py', '--cfg-path', '/tmp/tmpyuj5170r']
[2023-10-19T12:42:58.480+0000] {standard_task_runner.py:85} INFO - Job 221: Subtask waiting_for_data
[2023-10-19T12:42:58.468+0000] {standard_task_runner.py:57} INFO - Started process 1102 to run task
[2023-10-19T12:42:58.616+0000] {task_command.py:416} INFO - Running <TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-05-02T00:00:00+00:00 [running]> on host 6f5df7fea6fd
[2023-10-19T12:42:58.891+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='copy_tranform_to_mongo' AIRFLOW_CTX_TASK_ID='waiting_for_data' AIRFLOW_CTX_EXECUTION_DATE='2021-05-02T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2021-05-02T00:00:00+00:00'
[2023-10-19T12:42:58.914+0000] {base.py:73} INFO - Using connection ID 'fs_default' for task execution.
[2023-10-19T12:42:58.915+0000] {filesystem.py:64} INFO - Poking for file /opt/in/tiktok_google_play_reviews.csv
[2023-10-19T12:42:58.915+0000] {filesystem.py:69} INFO - Found File /opt/in/tiktok_google_play_reviews.csv last modified: 20220405221612
[2023-10-19T12:42:58.915+0000] {base.py:287} INFO - Success criteria met. Exiting.
[2023-10-19T12:42:58.943+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=copy_tranform_to_mongo, task_id=waiting_for_data, execution_date=20210502T000000, start_date=20231019T124258, end_date=20231019T124258
[2023-10-19T12:42:59.029+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-10-19T12:42:59.105+0000] {taskinstance.py:2778} INFO - 1 downstream tasks scheduled from follow-on schedule check
